frontend/frontendAngular/src/app/guards/login.guard.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot, UrlTree } from '@angular/router';
+import { Observable } from 'rxjs';
+import { LocalStorageLoginService } from '../services/local-storage-login.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginGuard implements CanActivate {
+  constructor(public localStorage: LocalStorageLoginService, private router: Router){}
+
+  canActivate(
+    route: ActivatedRouteSnapshot,
+    state: RouterStateSnapshot): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
+    if(!this.localStorage.logged){
```

Comment:
creio que conseguimos usar o get que voces mesmos criaram pra verificar se o usuario ta logado ou nao... e remover o logged to service... ou fazer o logged do service usar o get